### DISCUSSION TITLE
Time integration scheme

### DISCUSSION THREAD

**Discussion by Richard-happy** [idaholab/moose#23732] [Q&A General] [answered]
Hello,I wonder know if there is any way to  solve PDEs with different Time integration schemes within a coupled problem.
For example, solving a heat conduction model with implicit-euler and  solving a linear elastic model with bdf2.

**Comment by GiudGiud** [comment, marked as answer]
We dont support that within the same solve (eg coupling everything in the same matrix)

but you could use a multiapps for this
https://mooseframework.inl.gov/moose/syntax/MultiApps/index.html

**Reply by Richard-happy** [reply]
Thank you
